pkg/reconciler/userconfig/userconfig.go

```diff
@@ -249,7 +216,7 @@ func (r *ReconcilerUserConfig) cacheDeployment(ctx context.Context, log logr.Log
 			return err
 		}
 	}
-	cache.Spec.Template.Spec.ServiceAccountName = v1alpha1.CacheDeploymentName
+	cache.Spec.Template.Spec.ServiceAccountName = "pipeline"
```

Comment:
ok yeah if the cache just "works" with the existing `pipeline` SA's permissions then a super big plus one on this @stuartwdouglas 

at least that is how I read https://github.com/redhat-appstudio/jvm-build-service/pull/275/commits/5a28bd3e2bfd37012dc6374b3f5693ba14a87bad

Reply:
and yeah the artifact cache log at https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/redhat-appstudio_jvm-build-service/275/pull-ci-redhat-appstudio-jvm-build-service-main-jvm-build-service-in-repo-e2e/1575359207354404864/artifacts/jvm-build-service-in-repo-e2e/gather-extra/artifacts/pods/jvm-build-service-test-namespace-9sjnl_jvm-build-workspace-artifact-cache-8695db84b-qdntx_jvm-build-workspace-artifact-cache.log looks clean to me :-)